native/src/render/objects/resources/texture2D.cpp

```diff
@@ -255,6 +261,9 @@ namespace nova {
 
         command_buffer.copyBufferToImage(buffer, image, vk::ImageLayout::eTransferDstOptimal, 1, &region);
 
+#pragma clang diagnostic push
+#pragma ide diagnostic ignored "OCDFAInspection"
```

Comment:
what...

Reply:
I don't like warnings, and this one is a false positive. I can remove it if you want

Reply:
Fixed